src/main/ApplicationImpl.cpp

```diff
@@ -709,6 +709,8 @@ ApplicationImpl::validateAndLogConfig()
     {
         if (mConfig.isUsingBucketListDB())
         {
+            // Tx meta column no longer supported in BucketListDB
+            mPersistentState->dropTxMetaIfExists(*mDatabase);
```

Comment:
I think you need to make this schema change in the same place than other schema changes so that people can run the `upgrade-db` (some people don't give the account that runs core sql access to update schema)

Reply:
Done